tests/Processor/SanitizeStacktraceProcessorTest.php

```diff
@@ -65,9 +68,12 @@ public function testProcessWithPreviousException()
             $this->client->captureException($exception);
         }
 
-        $this->processor->process($this->client->_pending_events[0]);
+        $_pending_events = new \ReflectionProperty($this->client, '_pending_events');
```

Comment:
PHPUnit provides an `assertAttributeEquals` method to check whether an attribute (even private) equals a given value